Build failure: obs-studio-plugins.obs-transition-table

## Steps To Reproduce

Steps to reproduce the behavior:

1. build *obs-studio-plugins.obs-transition-table*

## Build log

<!-- insert build log in code block in collapsable section -->

<details>

<summary>Build Log</summary>

```bash
~ 
:: nix-shell -p obs-studio-plugins.obs-transition-table
this derivation will be built:
  /nix/store/a7dp7m9qlyr1va7bw10wb4dx4zgg5kjp-obs-transition-table-0.2.7.drv
building '/nix/store/a7dp7m9qlyr1va7bw10wb4dx4zgg5kjp-obs-transition-table-0.2.7.drv'...
Running phase: qtPreHook
Running phase: unpackPhase
unpacking source archive /nix/store/l19l4x4ca8vv221m2l3j24zid3ira6yv-source
source root is source
Running phase: patchPhase
Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/2yk77z2msxwbmqb9b26ybv5l23bv6f21-obs-transition-table-0.2.7/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/2yk77z2msxwbmqb9b26ybv5l23bv6f21-obs-transition-table-0.2.7/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/2yk77z2msxwbmqb9b26ybv5l23bv6f21-obs-transition-table-0.2.7/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/2yk77z2msxwbmqb9b26ybv5l23bv6f21-obs-transition-table-0.2.7/share/doc/transition-table -DCMAKE_INSTALL_INFODIR=/nix/store/2yk77z2msxwbmqb9b26ybv5l23bv6f21-obs-transition-table-0.2.7/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/2yk77z2msxwbmqb9b26ybv5l23bv6f21-obs-transition-table-0.2.7/share/man -DCMAKE_INSTALL_OLDINCLUDEDIR=/nix/store/2yk77z2msxwbmqb9b26ybv5l23bv6f21-obs-transition-table-0.2.7/include -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/2yk77z2msxwbmqb9b26ybv5l23bv6f21-obs-transition-table-0.2.7/include -DCMAKE_INSTALL_SBINDIR=/nix/store/2yk77z2msxwbmqb9b26ybv5l23bv6f21-obs-transition-table-0.2.7/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/2yk77z2msxwbmqb9b26ybv5l23bv6f21-obs-transition-table-0.2.7/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/2yk77z2msxwbmqb9b26ybv5l23bv6f21-obs-transition-table-0.2.7/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/3ix5h74n7ar9950vwzp4dxmil70pmx0k-gcc-wrapper-13.3.0/bin/strip -DCMAKE_RANLIB=/nix/store/3ix5h74n7ar9950vwzp4dxmil70pmx0k-gcc-wrapper-13.3.0/bin/ranlib -DCMAKE_AR=/nix/store/3ix5h74n7ar9950vwzp4dxmil70pmx0k-gcc-wrapper-13.3.0/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/2yk77z2msxwbmqb9b26ybv5l23bv6f21-obs-transition-table-0.2.7 -DBUILD_OUT_OF_TREE=On
-- The C compiler identification is GNU 13.3.0
-- The CXX compiler identification is GNU 13.3.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/3ix5h74n7ar9950vwzp4dxmil70pmx0k-gcc-wrapper-13.3.0/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/3ix5h74n7ar9950vwzp4dxmil70pmx0k-gcc-wrapper-13.3.0/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Performing Test HAVE_STDATOMIC
-- Performing Test HAVE_STDATOMIC - Success
-- Qt version found: 6
-- Found OpenGL: /nix/store/ymhcg6x5jrw3hx8ik1cji6awiybgp9f7-libglvnd-1.7.0/lib/libOpenGL.so
-- Found WrapOpenGL: TRUE
-- Found XKB: /nix/store/i0d0ws5xwix1dvmwp8w1fr4kz61k11x4-libxkbcommon-1.7.0/lib/libxkbcommon.so (Required is at least version "0.5.0")
-- Found WrapVulkanHeaders: /nix/store/1580k6qby7r99aywz4yad0in1yr33c9i-vulkan-headers-1.3.296.0/include
-- Configuring done (1.1s)
-- Generating done (0.0s)
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_EXPORT_NO_PACKAGE_REGISTRY
    CMAKE_POLICY_DEFAULT_CMP0025


-- Build files have been written to: /build/source/build
cmake: enabled parallel building
cmake: enabled parallel installing
Running phase: buildPhase
build flags: -j16 SHELL=/nix/store/0irlcqx2n3qm6b1pc9rsd2i8qpvcccaj-bash-5.2p37/bin/bash
[  0%] Built target transition-table_autogen_timestamp_deps
[ 20%] Automatic MOC and UIC for target transition-table
[ 20%] Built target transition-table_autogen
[ 60%] Building CXX object CMakeFiles/transition-table.dir/transition-table.cpp.o
[ 60%] Building CXX object CMakeFiles/transition-table.dir/transition-table_autogen/mocs_compilation.cpp.o
/build/source/transition-table.cpp: In constructor 'TransitionTableDialog::TransitionTableDialog(QMainWindow*)':
/build/source/transition-table.cpp:549:39: error: 'void QCheckBox::stateChanged(int)' is deprecated: Use checkStateChanged() instead [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Werror=deprecated-declarations8;;]
  549 |         connect(checkbox, &QCheckBox::stateChanged,
      |                                       ^~~~~~~~~~~~
In file included from /nix/store/w9g781rqrc12870vji13hycwh47lwsgd-qtbase-6.8.0/include/QtWidgets/QCheckBox:1,
                 from /build/source/transition-table.hpp:4,
                 from /build/source/transition-table.cpp:2:
/nix/store/w9g781rqrc12870vji13hycwh47lwsgd-qtbase-6.8.0/include/QtWidgets/qcheckbox.h:41:10: note: declared here
   41 |     void stateChanged(int);
      |          ^~~~~~~~~~~~
cc1plus: all warnings being treated as errors
make[2]: *** [CMakeFiles/transition-table.dir/build.make:97: CMakeFiles/transition-table.dir/transition-table.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:86: CMakeFiles/transition-table.dir/all] Error 2
make: *** [Makefile:156: all] Error 2
error: builder for '/nix/store/a7dp7m9qlyr1va7bw10wb4dx4zgg5kjp-obs-transition-table-0.2.7.drv' failed with exit code 2;
       last 25 log lines:
       > 
       > -- Build files have been written to: /build/source/build
       > cmake: enabled parallel building
       > cmake: enabled parallel installing
       > Running phase: buildPhase
       > build flags: -j16 SHELL=/nix/store/0irlcqx2n3qm6b1pc9rsd2i8qpvcccaj-bash-5.2p37/bin/bash
       > [  0%] Built target transition-table_autogen_timestamp_deps
       > [ 20%] Automatic MOC and UIC for target transition-table
       > [ 20%] Built target transition-table_autogen
       > [ 60%] Building CXX object CMakeFiles/transition-table.dir/transition-table.cpp.o
       > [ 60%] Building CXX object CMakeFiles/transition-table.dir/transition-table_autogen/mocs_compilation.cpp.o
       > /build/source/transition-table.cpp: In constructor 'TransitionTableDialog::TransitionTableDialog(QMainWindow*)':
       > /build/source/transition-table.cpp:549:39: error: 'void QCheckBox::stateChanged(int)' is deprecated: Use checkStateChanged() instead [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Werror=deprecated-declarations8;;]
       >   549 |         connect(checkbox, &QCheckBox::stateChanged,
       >       |                                       ^~~~~~~~~~~~
       > In file included from /nix/store/w9g781rqrc12870vji13hycwh47lwsgd-qtbase-6.8.0/include/QtWidgets/QCheckBox:1,
       >                  from /build/source/transition-table.hpp:4,
       >                  from /build/source/transition-table.cpp:2:
       > /nix/store/w9g781rqrc12870vji13hycwh47lwsgd-qtbase-6.8.0/include/QtWidgets/qcheckbox.h:41:10: note: declared here
       >    41 |     void stateChanged(int);
       >       |          ^~~~~~~~~~~~
       > cc1plus: all warnings being treated as errors
       > make[2]: *** [CMakeFiles/transition-table.dir/build.make:97: CMakeFiles/transition-table.dir/transition-table.cpp.o] Error 1
       > make[1]: *** [CMakeFiles/Makefile2:86: CMakeFiles/transition-table.dir/all] Error 2
       > make: *** [Makefile:156: all] Error 2
       For full logs, run 'nix log /nix/store/a7dp7m9qlyr1va7bw10wb4dx4zgg5kjp-obs-transition-table-0.2.7.drv'.
```

</details>

## Additional context

<!-- Add any other context about the problem here. -->

## Metadata

<!-- Please insert the output of running `nix-shell -p nix-info --run "nix-info -m"` below this line -->
`:: nix-shell -p nix-info --run "nix-info -m"`:
 - system: `"x86_64-linux"`
 - host os: `Linux 6.11.10-xanmod1, NixOS, 25.05 (Warbler), 25.05beta713604.fa42b5a5f401`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.10`
 - channels(root): `"home-manager, nixos, nur"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

## Notify maintainers

@flexiondotorg

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/applications/video/obs-studio/plugins/obs-transition-table.nix
+++ b/pkgs/applications/video/obs-studio/plugins/obs-transition-table.nix
@@ -9,23 +9,19 @@
 
 stdenv.mkDerivation rec {
   pname = "obs-transition-table";
-  version = "0.2.7";
+  version = "0.2.7-unstable-2024-11-27";
 
   src = fetchFromGitHub {
     owner = "exeldro";
     repo = "obs-transition-table";
-    rev = version;
-    sha256 = "sha256-rGF7hugC5ybpZBAIIXDiy3YDooMawf/yYX2YucQm2/U=";
+    rev = "976fe236dac7082b6c953f950fcb9e50495ce624";
+    sha256 = "sha256-TPRqKjEXdvjv+RfHTaeeO4GHur2j/+onehcu0I/HdD0=";
   };
 
   nativeBuildInputs = [ cmake ];
   buildInputs = [
     obs-studio
     qtbase
-  ];
-
-  cmakeFlags = [
-    "-DBUILD_OUT_OF_TREE=On"
   ];
 
   dontWrapQtApps = true;

